pom.xml

```diff
@@ -15,6 +15,7 @@
     <sonar.apiVersion>7.9</sonar.apiVersion>
     <jdk.min.version>1.8</jdk.min.version>
     <sonar.sources>src/main/java,src/main/js</sonar.sources>
+    <sonar.tests>src/test/java</sonar.tests>
```

Comment:
Should not be needed. `sonar.sources` is redefined only to add folder `src/main/js` which is not a default Maven source folder.